### PR TITLE
windows/mkdir: add page

### PR DESCRIPTION
**Pull request by sbrl**
For #1579

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/windows/mkdir.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# mkdir
+
+> Creates a directory.
+
+- Create a directory:
+
+`mkdir {{directory_name}}`
+
+- Recursively create a nested direcotry tree:
```
Comment (pxgamer) [inline pages/windows/mkdir.md]: Typo in directory. :+1:

File: pages/windows/mkdir.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+`mkdir {{path/to/sub_directory_name}}`
```
Comment (agnivade) [inline pages/windows/mkdir.md]: There seems to be no difference between the first and second command. Is this correct ?

File: pages/windows/mkdir.md (hunk 1 above)
Comment (sbrl) [reply pages/windows/mkdir.md]: Good spot, thanks!

File: pages/windows/mkdir.md (hunk 2 above)
Comment (sbrl) [reply pages/windows/mkdir.md]: Yep. I wanted to show explicitly that the windows variant of `mkdir` recursively creates directories by default, whereas the Linux one doesn't

**Pull request merged**